blazor/dropdown-list/filtering.md

```diff
@@ -7,103 +7,116 @@ control: DropDown List
+  EndsWith     | Checks whether a value ends with specified value.
+  Contains      | Checks whether a value contains with specified value.
+
+In the following example, `EndsWith` filter type has been mapped to the `FilterType` property.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/filter-type.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with Filter Type](./images/filtering/blazor_dropdown_filter-type.png)
+
+## Case sensitive filtering
+
+The Data items can be filtered either with or without case sensitivity using the [DataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.SfDataManager.html). This can be done by passing the fourth optional parameter [IgnoreCase](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.WhereFilter.html#Syncfusion_Blazor_Data_WhereFilter_IgnoreCase) of the [Where clause](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.Query.html#Syncfusion_Blazor_Data_Query_Where_Syncfusion_Blazor_Data_WhereFilter_).
+
+The following example shows how to perform case-sensitive filter.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/case-sentitive.razor %}
+
+{% endhighlight %}
+
+## Filter textbox placeholder 
+
+You can use [FilterBarPlaceholder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_FilterBarPlaceholder) to accept the value to be displayed as a watermark text on the filter bar.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/filter-textbox-placeholder.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with Filter Textbox Placeholder](./images/filtering/blazor_dropdown_filter-textbox-placeholder.png)
+
+## Custom filtering
+
+DropDownList component filter queries can be customized. You can also use your own filter libraries to filter data like Fuzzy search.
+
+In the below sample demonstration, filter the data either using its `FirstName` or `LastName` field. Hence in the Filtering event, [Predicate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.WhereFilter.html#Syncfusion_Blazor_Data_WhereFilter_Condition) is used with `or` condition for filtering both the fields. 
+
+For instance , the data source item consist of `FirstName` as `Nancy` and `LastName` as `Davalio`. But you can filter the data either by typing `N` or `D` character and it will showcase the `Nancy` (FirstName field) in the popup.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/case-sentitive.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with custom filtering](./images/filtering/blazor_dropdown_custom-filtering.png)
+
+## Multi column filtering 
+
+Provided a multi column style class in the built-in Syncfusion Blazor theme files. So, need to provide the multicolumn root class API name as `e-multi-column` in the CssClass property.
```

Comment:
We need to explain how we achieve filtering in multiple columns? like using Predicate.